src/uncontrast/pages/UploadNewPaper/ExperimentsSection/ConsciousnessMeasures.jsx

```diff
@@ -114,22 +134,216 @@ export default function ConsciousnessMeasures({
+                            " Indicate how many trials were taken for the objective awareness measure"
+                          }
+                        />
+                      </div>
+                    </div>
+                    <div className="flex gap-2 w-full items-center">
+                      <div className="w-3/4">
+                        <Text weight={"bold"} color={"grayReg"}>
+                          Number of participants in awareness test
+                        </Text>
+                      </div>
+                      <div className="w-1/4 flex justify-between items-center gap-2">
+                        <input
+                          min={0}
+                          disabled={fieldValues[index].id}
+                          type="number"
+                          value={fieldValue.awarenessTestedNum}
+                          onChange={(e) => {
+                            setFieldValues((prev) =>
+                              prev.map((item, i) =>
+                                i === index
+                                  ? {
+                                      ...item,
+                                      awarenessTestedNum: e.target.value,
+                                    }
+                                  : item
+                              )
+                            );
+                          }}
+                          onBlur={() =>
+                            submitCondition(index) &&
+                            handleSubmit(fieldValues, index)
+                          }
+                          onKeyDown={(e) =>
+                            e.key === "Enter" &&
+                            submitCondition(index) &&
+                            handleSubmit(fieldValues, index)
+                          }
+                          className={`border w-full border-gray-300 rounded-md p-2 ${
+                            fieldValues[index].id &&
+                            "bg-grayDisable text-gray-400"
+                          } `}
+                        />
+                        <TooltipExplanation isHeadline tooltip={" ?"} />
+                      </div>
+                    </div>
+                    <div className="flex gap-2 w-full items-center">
+                      <div className="w-3/4">
+                        <Text weight={"bold"} color={"grayReg"}>
+                          Is cm same participants as task?
+                        </Text>
+                      </div>
+                      <div className="w-1/4 flex justify-between items-center gap-2">
+                        <CustomSelect
+                          disabled={fieldValue?.id}
+                          value={fieldValue.is_cm_same_participants_as_task}
+                          onChange={(value) => {
+                            const newArray = [...fieldValues];
+                            newArray[index].is_cm_same_participants_as_task =
```

Comment:
The destructoring good. but notice it will still mutate the object inside the array